archspec/cpu/cpuid_flags.json

```diff
@@ -0,0 +1,622 @@
+{
+  "fpu": {
+    "eax": 1,
+    "edx": 0
+  },
```

Comment:
I changed the format of the JSON in #150 . This one has duplicate keys (for `eax=0x1` and `eax=0x80000001`). I also moved other constants we need in the JSON file.